src/main/java/duke/exception/InvalidArgumentException.java

```diff
@@ -0,0 +1,53 @@
+package duke.exception;
```

Comment:
I like how there's many ways that your code handles many different type of exception!